.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,78 @@
+name: Benchmarking
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  benchmarks:
+    name: Gate benchmarks
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+
+    steps:
+
+      - name: Checkout PennyLane-Lightning master
+        uses: actions/checkout@v2
+        with:
+          path: main
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install dependencies
+        run: sudo apt-get -y -q install cmake gcc
+
+      - name: Get required Python packages
+        run: |
+          cd main
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install matplotlib
+
+      - name: Install lightning.qubit device (master)
+        run: |
+          cd main
+          pip install -e .
+
+      - name: Benchmark lightning master device
+        run: |
+          cd main
+          python .github/workflows/benchmarks/run_bench.py lightning.qubit lightning_master.json
+
+      - name: Benchmark default qubit device
+        run: |
+          cd main
+          python .github/workflows/benchmarks/run_bench.py default.qubit default_qubit.json
+
+      - name: Checkout PennyLane-Lightning PR
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          path: main/pr
+
+      - name: Install lightning.qubit device (PR)
+        run: |
+          cd main/pr
+          pip install -e .
```

Comment:
I don't think so, but is there any need to be cautious and do `make clean` and `pip uninstall pennylane-lightning -y`?

Reply:
Sure! Adding these